src/stepfunctions/steps/utils.py

```diff
@@ -45,13 +49,19 @@ def get_aws_partition():
     return cur_partition
 
 
-"""
-ARN builder for task integration
-"""
-def resource_integration_arn_builder(service, api, integration_pattern=None):
+def get_service_integration_arn(service, api, integration_pattern=IntegrationPattern.RequestResponse):
```

Comment:
Move this to `integration_resources.py`. Having an unfocused `utils` module is usually an antipattern. You created a new module for service integration resources and this method definitely belongs there.

Reply:
Makes sense. Will do.